cc_hooks_gentoo.py

```diff
@@ -697,7 +697,7 @@ def pre_configure_hook(self, *args, **kwargs):
         if (ec['name'],ec['version']) in [('ROOT','5.34.36'), ('mariadb', '10.4.11')]:
             pass
         else:
-            update_opts(ec, ' -DCMAKE_SKIP_INSTALL_RPATH=ON ', 'configopts', PREPEND)
+            update_opts(ec, ' -DCMAKE_SKIP_INSTALL_RPATH=ON -DENABLE_XHOST=OFF ', 'configopts', PREPEND)
```

Comment:
Isn't this an option that is specific to PSI4 ? 

If so, the `configopts` should be modified in the `opts_changes` dictionary

Reply:
Some software actually use this flag. PSI4, chem, libxc

Reply:
Right, but this is something defined by those software, not something defined by CMake. `CMAKE_SKIP_INSTALL_RPATH` is something built into CMake

Reply:
In the case of a software that do not use it, CMake would simply warn about not using it. I wanted to avoid setting it per software in the hooks.

Reply:
I guess it 'probably' can't hurt...